### PR TITLE
Update legacy chain service info

### LINKED ISSUE #147: Update legacy chain service
### Description

Add the following information regarding legacy accounts:

You should be able to find the entire transaction history for your account (starting from the v1 genesis block until the v4 snapshot height) if you use the following link:

- For unintialized accounts: `https://legacy.lisk.com/histories/<legacy-address-ending-with-L>.csv`
- For initialized accounts: If not migrated: `https://legacy.lisk.com/histories/<lsk-address>.csv`

If the legacy account was initialized, the entire history including that of the corresponding legacy address will be consolidated under the lsk address and the *******L address's history won't be available separately.

### Location

https://docs.lisk.com/lisk-tools/utilities#legacy-chain-service

### Additional Information
https://discord.com/channels/405002561775599619/1119321332082024469/1250697341854552067

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #147

### CODE REVIEW

File: docs/lisk-tools/utilities.md [hunk 1]
```diff
@@ -53,9 +53,12 @@ How to use the legacy chain service:
 - Get a **transaction by id** `https://legacy.lisk.com/transactions/<TRANSACTION_ID>.json`
 - Get an **account by address** `https://legacy.lisk.com/accounts/<LEGACY_ACCOUNT_ADDRESS>.json`
 - Get **histories by address** `https://legacy.lisk.com/histories/<LEGACY_ACCOUNT_ADDRESS>.csv`
+
+  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy address `lsk*******`, or `*******L` for unitialized accounts.
  
-  This serves for all block histories in the past from height 1 https://legacy.lisk.com/blocks/1.json
+  If the legacy account was initialized, the entire history including that of the corresponding `*******L` legacy address will be consolidated under the lsk address and the `*******L` address's history won't be available separately.
```
Comment (sameersubudhi) [inline docs/lisk-tools/utilities.md]: Could we please also add a tool tip or note explaining what do we mean by `initialized`?
Basically, it means that the legacy account either had an outgoing transaction on Lisk Core v2. If not, it had performed a legacy reclaim transaction on Lisk Core v3/v4, prior to the L2 migration (perhaps specify the mainnet snapshot height).

Comment (Tschakki) [reply docs/lisk-tools/utilities.md]: Good idea, I added an info box.

**Review by sameersubudhi — APPROVED** [review]

File: docs/lisk-tools/utilities.md [hunk 2]
```diff
@@ -53,9 +53,16 @@ How to use the legacy chain service:
...
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/utilities.md]:
```suggestion
  If the legacy account was initialized, the entire history including that of the corresponding `*******L` legacy address will be consolidated under the `lsk` address and the `*******L` address's history won't be available separately.
```

File: docs/lisk-tools/utilities.md [hunk 3]
```diff
@@ -53,9 +53,16 @@ How to use the legacy chain service:
 - Get a **transaction by id** `https://legacy.lisk.com/transactions/<TRANSACTION_ID>.json`
 - Get an **account by address** `https://legacy.lisk.com/accounts/<LEGACY_ACCOUNT_ADDRESS>.json`
 - Get **histories by address** `https://legacy.lisk.com/histories/<LEGACY_ACCOUNT_ADDRESS>.csv`
+
+  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy address `lsk*******`, or `*******L` for unitialized accounts.
```
Comment (TalhaMaliktz) [inline docs/lisk-tools/utilities.md]: @sameersubudhi `lsk*******` is a legacy address? or is it just the binary address `*******L` that is a legacy address?
```suggestion
  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy address `*******L` for unitialized accounts.
```

Comment (sameersubudhi) [reply docs/lisk-tools/utilities.md]: @TalhaMaliktz Yes! They are both legacy addresses now because they were part of Lisk L1 format. Now, all the Lisk addresses on L2 are ETH compliant and are 20-byte long hex strings of the format `0x****************************************`.
```suggestion
  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy Lisk address of the format:`lsk**************************************` or `*******************L` (for uninitialized accounts).
```

File: docs/lisk-tools/utilities.md [hunk 4]
```diff
@@ -53,9 +53,16 @@ How to use the legacy chain service:
...
  
-  This serves for all block histories in the past from height 1 https://legacy.lisk.com/blocks/1.json
+  If the legacy account was initialized, the entire history including that of the corresponding `*******L` legacy address will be consolidated under the lsk address and the `*******L` address's history won't be available separately.
+
+  For unintialized accounts you can find the account history under: `https://legacy.lisk.com/histories/<legacy-address-ending-with-L>.csv`
```
Comment (sameersubudhi) [inline docs/lisk-tools/utilities.md]: BTW do we need this statement? We already specify the necessary information on L57.
```suggestion
  For uninitialized accounts you can find the account history under: `https://legacy.lisk.com/histories/<legacy-address-ending-with-L>.csv`
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: docs/lisk-tools/utilities.md (hunk 4 above)
Comment (Tschakki) [reply docs/lisk-tools/utilities.md]: Yes, I think we can remove it 👍 
```suggestion
```

File: docs/lisk-tools/utilities.md [hunk 5]
```diff
@@ -53,9 +53,15 @@ How to use the legacy chain service:
 - Get a **transaction by id** `https://legacy.lisk.com/transactions/<TRANSACTION_ID>.json`
 - Get an **account by address** `https://legacy.lisk.com/accounts/<LEGACY_ACCOUNT_ADDRESS>.json`
 - Get **histories by address** `https://legacy.lisk.com/histories/<LEGACY_ACCOUNT_ADDRESS>.csv`
+
+  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy Lisk address of the format:`lsk**************************************` or `*******************L` (for uninitialized accounts).
  
-  This serves for all block histories in the past from height 1 https://legacy.lisk.com/blocks/1.json
+  If the legacy account was initialized, the entire history including that of the corresponding `*******L` legacy address will be consolidated under the `lsk` address and the `*******L` address's history won't be available separately.
```
Comment (sameersubudhi) [inline docs/lisk-tools/utilities.md]:
```suggestion
  If a Lisk L1 legacy account (`*******************L` address) was initialized, the entire history including that of the `*******************L` legacy address will be consolidated under the corresponding `lsk` address. The `*******************L` address's history won't be available separately.
```

File: docs/lisk-tools/utilities.md [hunk 6]
```diff
@@ -53,9 +53,15 @@ How to use the legacy chain service:
 - Get a **transaction by id** `https://legacy.lisk.com/transactions/<TRANSACTION_ID>.json`
 - Get an **account by address** `https://legacy.lisk.com/accounts/<LEGACY_ACCOUNT_ADDRESS>.json`
 - Get **histories by address** `https://legacy.lisk.com/histories/<LEGACY_ACCOUNT_ADDRESS>.csv`
+
+  where `<LEGACY_ACCOUNT_ADDRESS>` is the legacy Lisk address of the format:`lsk**************************************` or `*******************L` (for uninitialized accounts).
```
Comment (sameersubudhi) [inline docs/lisk-tools/utilities.md]: To make it less confusing for the users, we could rephrase it to:
```suggestion
  where `<LEGACY_ACCOUNT_ADDRESS>` is the Lisk L1 address of the format:`lsk**************************************` or `*******************L` (for uninitialized accounts).
```

File: docs/lisk-tools/utilities.md [hunk 7]
```diff
@@ -53,9 +53,15 @@ How to use the legacy chain service:
...
  
-  This serves for all block histories in the past from height 1 https://legacy.lisk.com/blocks/1.json
+  If the legacy account was initialized, the entire history including that of the corresponding `*******L` legacy address will be consolidated under the `lsk` address and the `*******L` address's history won't be available separately.
+
 
+:::info[INFO: When is an account initialized?]
+A legacy account is initialized, if it either has an outgoing transaction on Lisk Core v2, or had performed a [legacy reclaim transaction](https://github.com/LiskArchive/lisk-core/blob/development/src/application/modules/legacy/commands/reclaim.ts) on Lisk Core v3/v4, prior to the Lisk L2 migration, which happened end of May 2024, at block height 24,823,618.
```
Comment (sameersubudhi) [inline docs/lisk-tools/utilities.md]:
```suggestion
:::info[INFO: When is a Lisk L1 account initialized?]
A Lisk L1 legacy account is considered initialized, if it either has an outgoing transaction on Lisk Core v2, or had performed a [legacy reclaim transaction](https://github.com/LiskArchive/lisk-core/blob/development/src/application/modules/legacy/commands/reclaim.ts) on Lisk Core v3/v4, prior to the Lisk L2 migration, which happened end of May 2024, at block height 24,823,618.
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
The previous statements could still confusing for an user. Attempted to make it clearer. Please have a look 🙏

**Review by TalhaMaliktz — COMMENTED** [review]
@Tschakki, I think we should move the Legacy Chain text out of this page and create a new one. I also suggest that the `Lisk L1->L2 migration guide`  should be moved to a new category as well. Following is the suggested structure. 

```
Building on Lisk

About Lisk L1 
 - Lisk L1->L2 migration guide
 - Legacy Chain Information.

Lisk Tools
```

I feel right now that the aforementioned info is kind of hidden and doesn't fit the current structure. To make it more visible for our L1 grant partners or people who are still trying to access data from the Legacy L1 chain, we should do this small move.

**Review by sameersubudhi — APPROVED** [review]
Thank you for the changes 🙏

**Review by TalhaMaliktz — APPROVED** [review]
Perfect 👌🏼! thank you 🎉
